packages/flutter/lib/src/material/time_picker.dart

```diff
@@ -1561,6 +1565,7 @@ class _HourMinuteTextField extends StatefulWidget {
     @required this.selectedTime,
     @required this.isHour,
     @required this.style,
+    @required this.semanticHintText,
```

Comment:
What's the rationale as to where this falls in the list of fields?

Reply:
I put it above the callbacks but below existing params since they're all of different types. It's a private class so the order isn't super important